library/x509.c

```diff
@@ -1503,10 +1503,16 @@ int mbedtls_x509_info_subject_alt_name(char **buf, size_t *size,
 
             /*
              * dNSName
+             * RFC822 Name
              */
             case MBEDTLS_X509_SAN_DNS_NAME:
+            case MBEDTLS_X509_SAN_RFC822_NAME:
             {
-                ret = mbedtls_snprintf(p, n, "\n%s    dNSName : ", prefix);
+                char * dns_name = "dNSName";
```

Comment:
This will fail code style checks, you want:
```c
char *dns_name = "dNSName";
```
CI is also failing because of a type conversion warning, so:
```c
const char *dns_name = "dNSName";
```